eng/common/pipelines/templates/steps/get-code-owners.yml

```diff
@@ -0,0 +1,24 @@
+parameters: 
```

Comment:
I don't see you using this template anywhere yet. Isn't this for the next stage when you want to get the ms alias?

Reply:
Sorry nevermind I see you aren't resolving the ms alias yet just the code owners but I don't see you calling it anywhere.

Reply:
This is being called for archetype-lang-release yml.
https://github.com/Azure/azure-sdk-for-python/blob/add_author/eng/pipelines/templates/stages/archetype-python-release.yml#L271